com.itopia.client.json

```diff
@@ -0,0 +1,245 @@
+                  "--disable-avfilter",
+                  "--disable-programs",
+                  "--disable-doc"
+              ],
+              "cleanup": [
+                  "/share/ffmpeg"
+              ],
+              "sources": [
+                  {
+                      "type": "git",
+                      "url": "https://github.com/FFmpeg/ffmpeg.git",
+                      "tag": "n4.2.1"
+                  }
+              ]
+          },
+          {
+              "name": "openh264",
+              "buildsystem": "simple",
+              "build-commands": [ "make -j PREFIX=/app install" ],
+              "sources": [
+                  {
+                      "type": "git",
+                      "url": "https://github.com/cisco/openh264.git",
+                      "tag": "v1.8.0"
+                  }
+              ]
+          },
+          {
+              "name": "freerdp",
+              "buildsystem": "cmake-ninja",
+              "config-opts": [
+                  "-DCMAKE_INSTALL_LIBDIR:PATH=lib",
+                  "-DCHANNEL_TSMF:BOOL=OFF",
+                  "-DCHANNEL_URBDRC:BOOL=ON",
+                  "-DCHANNEL_PRINTER:BOOL=ON",
+                  "-DCHANNEL_PRINTER_CLIENT:BOOL=ON",
+                  "-DBUILD_TESTING:BOOL=OFF",
+                  "-DWITH_MANPAGES:BOOL=OFF",
+                  "-DWITH_GSSAPI:BOOL=OFF",
+                  "-DWITH_PCSC:BOOL=ON",
+                  "-DWITH_SERVER:BOOL=OFF",
+                  "-DWITH_SAMPLE:BOOL=OFF",
+                  "-DWITH_CUPS:BOOL=OFF",
+                  "-DWITH_OPENH264:BOOL=ON",
+                  "-DWITH_OPENH264_LOADING:BOOL=OFF",
+                  "-DWITH_FFMPEG:BOOL=OFF",
+                  "-DWITH_DSP_FFMPEG:BOOL=OFF",
+                  "-DWITH_OSS:BOOL=OFF",
+                  "-DWITH_PULSE:BOOL=ON",
+                  "-DWITH_ALSA:BOOL=OFF",
+                  "-DWITH_OSS:BOOL=OFF",
+                  "-DWITH_CAIRO:BOOL=ON",
+                  "-DWITH_SWSCALE:BOOL=ON",
+                  "-DWITH_WINPR_TOOLS:BOOL=OFF",
+                  "-DWITH_WAYLAND:BOOL=OFF",
+                  "-DWITH_CLIENT:BOOL=OFF"
+              ],
+              "sources": [
+                  {
+                      "type": "git",
+                      "url": "https://github.com/FreeRDP/FreeRDP.git",
+                      "branch": "master"
+                  }
+              ]
+          },
+          {
+              "name": "client",
+              "buildsystem": "simple",
+              "cleanup": [
+                  "client.tar.xz"
+              ],
+              "build-commands": [
+                  "tar xf client.tar.xz --strip-components=1 -C /app"
+              ],
+              "sources": [
+                  {
+                      "type": "file",
+                      "dest-filename": "client.tar.xz",
+                      "url": "https://storage.googleapis.com/rdp-client/linux/versions/itopia-client-v0.9.14.tar.xz",
```

Comment:
Does it work on non-x86_64 CPU architectures? If not, you need to limit the build to x86_64. https://github.com/flathub/flathub/wiki/App-Maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
@barthalion out of curiosity, how can you specify architecture specific sources here? e.g. one zip per architecture

Reply:
@akallabeth see the example https://github.com/flathub/org.freedesktop.Sdk.Extension.golang/blob/86dd1fc2f022152e7caf6a888854a7592c358231/org.freedesktop.Sdk.Extension.golang.yaml#L10